scaffolds/hello-world-vue/scaffold.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Template, Zombi } from 'compiled/zombi';
+import { createScaffold } from 'cli/utils/scaffold-helpers';
+import { mergePrompts } from 'cli/utils/merge-prompts';
+import { NpmClientPrompt, PublishableApiKeyPrompt } from 'scaffolds/prompts';
+
+type HelloWorldVueData = NpmClientPrompt.Data & PublishableApiKeyPrompt.Data;
+
+export default createScaffold<HelloWorldVueData>(
+  (props) => (
+    <Zombi {...props} prompts={mergePrompts(PublishableApiKeyPrompt.questions, NpmClientPrompt.questions)}>
+      <Template source="./src" />
+      <Template source="./.env" />
+      <Template source="./.gitignore" />
+      <Template source="./.eslintrc.js" />
+      <Template source="./babel.config.js" />
+      <Template source="./package.json" />
+      <Template source="./README.md" />
+      <Zombi {...props} data={false}>
+        <Template source="./public" />
+      </Zombi>
```

Comment:
We can actually skip the nested `<Zombi>` here, the `<Template />` component also accepts a `data` prop. So we can do this:

```tsx
<Template source="./public" data={false} />
```

Reply:
Also, just for your knowledge: the order of `data` precedence is `<Template>` first, then `<Zombi>` as a fallback.

Reply:
Fixed.